### PR TITLE
Updating Stemcell path in documentation.

### PR DESCRIPTION
**Pull request by jesseproudman**
Ensuring docs are using the new path.

### CODE REVIEW

**Comment by jesseproudman** [comment]
Submitting Blue Box CLA tomorrow.

File: source/docs/running/deploying-cf/openstack/install_cf_openstack.html.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ A cloud provider needs a base image to provision VMs/servers. Bosh explicitly re
 To upload the latest bosh stemcell to your bosh:
 
 <pre class="terminate">
-$ wget http://bosh-jenkins-artifacts.s3.amazonaws.com/bosh-stemcell/openstack/latest-bosh-stemcell-openstack.tgz
+$ wget http://bosh-jenkins-artifacts.s3.amazonaws.com/bosh-stemcell/openstack/bosh-stemcell-latest-openstack-kvm-ubuntu.tgz
```
Comment (andreasmaier) [inline source/docs/running/deploying-cf/openstack/install_cf_openstack.html.md]: We spoke with @tsaleh about this and the latest-bosh-stemcell-openstack.tgz url is coming back within a week. We'd like to keep that as the URL going forward. The rest of the changes look good to merge.

@andreasmaier / @mmb

Comment (tammersaleh) [inline source/docs/running/deploying-cf/openstack/install_cf_openstack.html.md]: Yeah, it was accidentally deleted.  Looks like we had an expiration policy on that bucket that we weren't aware of.

**Comment by jesseproudman** [comment]
Updated and reverted back the URL.

**Comment by jesseproudman** [comment]
Submitting a new PR using actual branch.

**Pull request closed without merging**